### PR TITLE
#167014104 Deploy app to heroku

### PR DESCRIPTION
**Pull request by sebbss**
#### What does this PR do?

Deploy the app to Heroku.

#### Description of Task to be completed?

- add static.json
- modify scripts for running the application
#### How should this be manually tested?
to view the Heroku deployment you can visit the heroku app through this [link](https://ah-frontend-space-staging.herokuapp.com/)

#### What are the relevant pivotal tracker stories?

[#167014104](https://www.pivotaltracker.com/story/show/167014104)

#### Screenshots (if appropriate)

image

### CODE REVIEW

**Review by frankopkusianwar — APPROVED** [review]
LGTM

**Review by NicholusMuwonge — CHANGES_REQUESTED** [review]
Please kindly add an empty line at the end of this file

File: app.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      {
+        "url": "heroku/nodejs"
+      },
+      {
+        "url": "https://github.com/heroku/heroku-buildpack-static"
+      }
+    ],
+    "stack": "heroku-18"
+  } 
+
```
Comment (richien) [inline app.json]: @sebbss , kindly add a new line to this file as well as to the `statis.json` file. Other than that, I have tested the deployed app and it works as described in the PR.

**Review by richien — CHANGES_REQUESTED** [review]

**Review by MuhanguziDavid — COMMENTED** [review]
Nice work with the deployment @sebbss , please ensure that your commit follows the Engineering playbook standards

**Review by richien — APPROVED** [review]
Nice work @sebbss.

**Review by NicholusMuwonge — APPROVED** [review]
Thanks for the excellent work @sebbss.

**Pull request merged**